src/main/java/hydroguard/api/mock/controllers/plants/PlantController.java

```diff
@@ -57,6 +57,7 @@ public PlantsDTO getAllPlants(
             @RequestParam(defaultValue = "1") int page,
             @RequestParam(defaultValue = "10") int pageSize,
             @RequestParam(required = false) String name,
+            @RequestParam(required = false) String householdId,
```

Comment:
why string is it not UUID

Reply:
Dat is een hele goede vraag, ask AI i guess. Zal ff aanpassen
![image](https://github.com/user-attachments/assets/6bb7ae17-e604-499b-b88b-39e87f2ddaae)
i hate AI sometimes